kotlin/internal/jvm/compile.bzl

```diff
@@ -772,7 +772,7 @@ def _run_kt_java_builder_actions(
         java_info = java_common.compile(
             ctx,
             source_files = srcs.java,
-            source_jars = generated_kapt_src_jars + srcs.src_jars,
+            source_jars = generated_kapt_src_jars + srcs.src_jars + generated_ksp_src_jars,
```

Comment:
Doesn't this essentially mean that all targets that execute KSP will be forced to run a Java compilation action even if the underlying KSP plugin only produces Kotlin or class file outputs?